build/includes/examples.mk

```diff
@@ -34,6 +35,7 @@ test-examples-on-gcr: example-image-test.crd-client
 test-examples-on-gcr: example-image-test.supertuxkart
 test-examples-on-gcr: example-image-test.simple-game-server
 
+push-example-golang-images: example-image-push.allocation-endpoint
```

Comment:
If you add this line here then you also need a push target in the new Makefile (which doesn't. exist).